cron/loginstats.go

```diff
@@ -0,0 +1,70 @@
+package cron
+
+import "database/sql"
+
+type Login struct {
+	Count int
+}
+
+// /logins/distinct
+func GetDistinctLoginCount(db *sql.DB, startDate string, endDate string) (Login, error){
+	var count int
+
+	query := `SELECT count(distinct user_id) FROM logins WHERE login_time >= ($1 :: DATE )
+		AND login_time <= ($2 :: DATE);`
+
+	rows, err := db.Query(query, startDate, endDate)
+
+	if err != nil {
+		return Login{-1}, err
+	}
+
+	defer rows.Close()
+	var logins Login
+	logins = Login{0}
```

Comment:
Since 0 is the zero value for an int, you don't need this line at all. It's already 0 due to the `var logins Login` line above.